tests/NHSD.BuyingCatalogue.API.IntegrationTests/NHSD.BuyingCatalogue.API.IntegrationTests.csproj

```diff
@@ -25,4 +25,8 @@
     <ProjectReference Include="..\NHSD.BuyingCatalogue.Testing.Tools\NHSD.BuyingCatalogue.Testing.Tools.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Annoyingly this gets added by specflow. Don't actually need this change.

Reply:
eugh thought I removed these